homeassistant/components/philips_js/light.py

```diff
@@ -261,7 +282,9 @@ def _handle_coordinator_update(self) -> None:
         self._update_from_coordinator()
         super()._handle_coordinator_update()
 
-    async def _set_ambilight_cached(self, algorithm, hs_color, brightness):
+    async def _set_ambilight_cached(
+        self, effect: AmbilightEffect, hs_color, brightness
+    ):
```

Comment:
If you type one parameter, please type the other too.

Reply:
Done